distribution/lib/Standard/Database/0.0.0-dev/src/Connection/Connection.enso

```diff
@@ -60,7 +60,7 @@ type Connection
 
        Arguments:
         - database: The name of the database to connect to.
-    set_database : Text -> Connection ! Sql_Error
+    set_database : Text -> Connection -> Connection ! Sql_Error
```

Comment:
Something doesn't seem right here.
`set_database` takes 1 argument: `database` and nothing else - so why the function type suggests it takes a `Connection`?

(I mean it does take `self` too, **but** we never included these in the signature and `self` comes before `database` whereas this `Connection` comes after `Text`.)

So I assume a typo?
```suggestion
    set_database : Text -> Connection ! Sql_Error
```